include/Ring.hh

```diff
@@ -54,6 +55,7 @@ class TiltedRing : public PropertyObject, public Buildable, public Identifiable<
 
  TiltedRing() :
   //materialObject_(MaterialObject::ROD),
+    moduleNode            ("Module"                , parsedOnly()),
```

Comment:
Out of my ignorance: why do you need to add Module here?

Reply:
Ah, o: I get it: to assign individual rhoShifts and twists, is that correct?

Reply:
Indeed - this is one of the parts that could do with improvement (both for readability of the config and for avoiding the construction 2 comments below)